specifications/api/swagger.yaml

```diff
@@ -117,6 +117,42 @@ x-numberOfBlocks: &numberOfBlocks
         - block
       example: "block"
 
+x-genesis: &genesis
+  type: object
+  required:
+    - unit
+  properties:
+    unit:
+      type: string
+      enum:
+        - genesis
+      example: "genesis"
+
+x-tip: &tip
```

Comment:
the same here. what about having here just string? not object? unit seems weird for me ...

Reply:
if we have field "restoration_mode" maybe it would be better to have string "from_tip"